File: beecrowd/cpp/p1015.cpp
// Distância Entre Dois Pontos

#include <bits/stdc++.h>

using namespace std;

int main()
{
    ios_base::sync_with_stdio(0);
    cin.tie(0);

    double x1, y1, x2, y2;
    cin >> x1 >> y1 >> x2 >> y2;
    cout << fixed << setprecision(4) << sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2)) << '\n';

    return 0;
}

File: beecrowd/python/p1015.py
# Distância Entre Dois Pontos

import sys
from math import sqrt

x1, y1, x2, y2 = map(float, sys.stdin.read().split())
dx = x2 - x1
dy = y2 - y1
sys.stdout.write(f"{sqrt(dx * dx + dy * dy):.4f}\n")
